src/main/resources/public/jsx/util/form.jsx

```diff
@@ -38,16 +41,20 @@ InputText.propTypes = {
     divClassName: React.PropTypes.string,
     error: React.PropTypes.bool,
     errorMsg: React.PropTypes.string,
-    isRequired: React.PropTypes.bool
+    isRequired: React.PropTypes.bool,
+    disabled: React.PropTypes.bool
 }
 
-const Select = ({ name, value, label, onChange, children, labelClassName, divClassName, error, errorMsg, isRequired }) =>
+const Select = ({ name, value, label, onChange, children, labelClassName, divClassName, error, errorMsg, isRequired, emptyValue }) =>
```

Comment:
Maybe using the isRequired prop should be enough to know if you want to display an empty default option